packages/babel/test/fixtures/with-query-and-hash/module.js

```diff
@@ -0,0 +1 @@
+export default class WithQueryAndHash {}
```

Comment:
This file only exist to let node-resolve plugin resolve module ID in test. This file content doesn't matter.
The actual file content is loaded by the `load` rollup hook defined in the test.